### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by tushar-max**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -3,7 +3,7 @@ README
 ----------------------------------------------------------------------
 
 Please refer to the README file within the doc-base/ directory. Your
-directory structure is probably like:
+directory structure is probably looks like:
```
Comment (cmb69) [inline README:6]:
```suggestion
directory structure probably looks like:
```
I don't see why that would have to be changed, though.

**Comment by tushar-max** [comment]
I've corrected grammar mistake

Please accept pull request
It's about winning a contest. I've logged in to github today only

**Comment by cmb69** [comment]
> It's about winning a contest.

No. It's about improving the docs.

**Comment by tushar-max** [comment]
I've corrected grammar

**Comment by salathe** [comment]
No, thanks.

**Pull request closed without merging**